Makefile

```diff
@@ -26,9 +26,16 @@ install: get-deps
 	go install ./...
 
 test: hdfs
+# Run unit tests only without Kerberos.
+ifndef KERBEROS
```

Comment:
Why skip all the unit tests? We can skip just the permission ones if necessary (with `t.Skip("skipping permission tests under kerberos")`)